ui/js/reactrevisions.jsx

```diff
@@ -4,9 +4,9 @@ const MoreRevisionsLink = (props) => (
     <li>
         <a href={props.href}
            data-ignore-job-clear-on-click={true}
-           target='_blank'>
```

Comment:
I presume these are required since previously we were accidentally not including .jsx files in the eslint run?